frame/staking/src/pallet/impls.rs

```diff
@@ -1309,6 +1309,70 @@ impl<T: Config> ScoreProvider<T::AccountId> for Pallet<T> {
 	}
 }
 
+/// A simple sorted list implementation that does not require any additional pallets. Note, this
+/// does not provide validators in sorted order. If you desire nominators in a sorted order take
+/// a look at [`pallet-bags-list].
+pub struct UseValidatorsMap<T>(sp_std::marker::PhantomData<T>);
```

Comment:
Some tests for this would be a nice addition.

Reply:
I am actually wondering why would we need any tests for this if it's used in mocks and as a stub. We might be better off just with those tests that mocks are there for if that makes any sense.